packages/neptune-extension/src/labextension/components/ToolbarButton.less

```diff
@@ -1,6 +1,10 @@
 .n-toolbar-button {
   min-width: 27px !important;
 
+  &__icon {
+    display: inline !important;
+  }
+
   &--logo {
     background-image: url('../../common/assets/logo.svg') !important;
```

Comment:
I think aliases works here as well, the following code should work
`background-image: url('~common/assets/logo.svg') !important;`